### DISCUSSION TITLE
MacOS: block cursor doesn't work, always display as a bar

### DISCUSSION THREAD

**Discussion by fascox** [ghostty-org/ghostty#4071] [Feature Requests, Ideas] [answered]
as per title. Same zsh shell configuration is ok under iTerm and native terminal, not in Ghostty.

**Comment by frobware** [comment, marked as answer]
I had to use `shell-integration-features = no-cursor` to keep the block cursor.

**Comment by fascox** [comment]
Awesome, thank you! It worked perfectly.
